contracts/src/teleporter/registry/TeleporterOwnerUpgradeable.sol

```diff
@@ -3,25 +3,31 @@
 
 // SPDX-License-Identifier: Ecosystem
 
-pragma solidity 0.8.18;
+pragma solidity 0.8.23;
 
 import {TeleporterUpgradeable} from "./TeleporterUpgradeable.sol";
-import {Ownable} from "@openzeppelin/contracts@4.8.1/access/Ownable.sol";
+import {OwnableUpgradeable} from
+    "@openzeppelin/contracts-upgradeable@5.0.2/access/OwnableUpgradeable.sol";
 
 /**
  * @dev Contract that inherits {TeleporterUpgradeable} and allows
  * only owners of the contract to update the minimum Teleporter version.
  *
  * @custom:security-contact https://github.com/ava-labs/teleporter/blob/main/SECURITY.md
  */
-abstract contract TeleporterOwnerUpgradeable is TeleporterUpgradeable, Ownable {
-    constructor(
+abstract contract TeleporterOwnerUpgradeable is TeleporterUpgradeable, OwnableUpgradeable {
+    // solhint-disable-next-line func-name-mixedcase
+    function __TeleporterOwnerUpgradeable_init(
         address teleporterRegistryAddress,
         address initialOwner
-    ) TeleporterUpgradeable(teleporterRegistryAddress) {
-        transferOwnership(initialOwner);
+    ) internal onlyInitializing {
+        __TeleporterUpgradeable_init(teleporterRegistryAddress);
+        __Ownable_init(initialOwner);
```

Comment:
Just confirming that the OZ standard is to not call `XXX_init_unchained()` at the end of `XXX_init()` if the unchained call is a no-op as it is here?

Doesn't make a functional difference, but want to make sure we're consistent with it going forward.

Reply:
they do not call it from `ContextUpgradeable`. But imo the OZ contract upgradeabls do not have many standards due to the limitation of customizability of their transpiler, so think we can set standards of our own. This would be a tradeoff for small gas increase for internal call, vs preventing future possibility of missing the `unchained` call if we add logic there. No strong pref

Reply:
Fine with leaving it as is. Any change that would make it possibly incorrect is a change to this contract itself (not just updating a version of a parent contract or the like).